Update redhat-create-upload-vhd.md

johanburati opened the pull request:
The raw image needs to be aligned on the megabyte boundary, the VHD is made of a raw disk image followed by a VHD footer (512 bytes), therefore if the raw image is aligned to a  megabyte boundary the VHD file cannot be.
 
If the raw image is not aligned on the megabyte boundary you will not be able to convert it to a managed disk, you will get a “Only blobs formatted as VHDs can be imported” error. If the VHD size is a multiple of 1 MB, that means the raw image is not and your will get the error.

azurecla commented:
Hi __@lju__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

johanburati committed:
Update redhat-create-upload-vhd.md

The raw image needs to be aligned on the megabyte boundary, the VHD is made of a raw disk image followed by a VHD footer (512 bytes), therefore if the raw image is aligned to a  megabyte boundary the VHD file cannot be.
 
If the raw image is not aligned on the megabyte boundary you will not be able to convert it to a managed disk, you will get a “Only blobs formatted as VHDs can be imported” error. If the VHD size is a multiple of 1 MB, that means the raw image is not and your will get the error.
